tutorials/ui/gui_using_fonts.rst

```diff
@@ -8,51 +8,727 @@ Godot allows you to set specific fonts for different UI nodes.
+    depends on the distribution. This means that on different Linux
+    distributions, different fonts may be displayed for a given system font name
+    or alias.
+
+.. _doc_using_fonts_font_prerendering:
+
+Font prerendering
+^^^^^^^^^^^^^^^^^
+
+When using traditional rasterized fonts, Godot caches glyphs on a per-font and
+per-size basis. This reduces stuttering, but it can still occur the first time a
+glyph is displayed when running the project. This can be especially noticeable
+at higher font sizes or on mobile devices.
+
+When using MSDF fonts, they only need to be rasterized once to a special signed
+distance field texture. This means caching can be done purely on a per-font
+basis, without taking the font size into consideration. However, the initial
+rendering of MSDF fonts is slower compared to a traditional rasterized font at a
+medium size.
+
+To avoid stuttering issues related to font rendering, it is possible to
+*prerender* certain glyphs. This can be done for all glyphs you intend to use
+(for optimal results), or only for common glyphs that are most likely to appear
+during gameplay (to reduce file size). Glyphs that aren't pre-rendered will be
+rasterizd on-the-fly as usual.
+
+.. note::
+
+    In both cases (traditional and MSDF), font rasterization is done on the CPU.
+    This means that the GPU performance doesn't affect how long it takes for fonts
+    to be rasterized.
+
+Open the Advanced Import Settings dialog by double-clicking the font file in the
+FileSystem dock. You can also select the font in the FileSystem dock, go to the
+Import dock then choose **Advanced…** at the bottom:
+
+.. figure:: img/using_fonts_advanced_import_settings.webp
+   :align: center
+
+   Import dock
+
+Move to the **Pre-render Configurations** tab of the Advanced Import Settings dialog,
+then add a configuration by clicking the "plus" symbol:
+
+.. figure:: img/using_fonts_advanced_import_settings_prerender_new_configuration.webp
+   :align: center
+   :alt: Adding a new prerendering configuration in the Advanced Import Settings dialog
+
+   Adding a new prerendering configuration in the Advanced Import Settings dialog
+
+After adding a configuration, make sure it is selected by clicking its name
+once. You can also rename the configuration by double-clicking it.
+
+There are 2 ways to add glyphs to be prerendered to a given configuration. It is
+possible to use both approaches in a cumulative manner:
+
+**Using text from translations**
+
+For most projects, this approach is the most convenient to use, as it
+automatically sources text from your language translations. The downwide is that
+it can only be used if your project supports
+:ref:`internationalization <doc_internationalizing_games>`. Otherwise, stick to
+the "Using custom text" approach described below.
+
+After adding translations to the Project Settings, use the
+**Glyphs from the Translations** tab to check translations by double-clicking them,
```

Comment:
I've noticed that you need to double-click translations to add them, rather than single-clicking them. This doesn't seem logical to me, since there is no action that could otherwise be done by single-clicking. cc @bruvzg